### DISCUSSION TITLE
How does Faust handle wp-cron?

### DISCUSSION THREAD

**Discussion by traed** [wpengine/faustjs#1853] [Q&A] [answered]
I can't find any mention of WP cron in the docs. Since it's normally run on page load, how is it affected by running WP headless?

**Comment by mindctrl** [comment, marked as answer]
@traed hey, the WP Cron runs when the site receives traffic, if the time is right. If the site gets no traffic of any kind, the cron typically can't run. Faust doesn't do anything special with WP Cron. But, if your site is using dynamic rendering, or is being statically built on occasion, the connections made to WP to do the render/build would cause the cron to run.

If it's not running consistently, you can use a third party service, a script, or something along those lines to hit `/wp-cron.php?doing_wp_cron` on a schedule to force WP to run it.

Hope this helps! Let me know if you have any questions.
